storage/tsdb/tsdb.go

```diff
@@ -117,6 +121,9 @@ type Options struct {
 	// The maximum timestamp range of compacted blocks.
 	MaxBlockDuration model.Duration
 
+	// The maximum size of each WAL segment files
```

Comment:
"files" -> "file"

nit: add period at end of comment.

Reply:
Done